daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -168,9 +168,22 @@ commandParser = subparser $ fold
         jsonApiOptions <- many (strOption (long "json-api-option" <> metavar "JSON_API_OPTION" <> help "Pass option to HTTP JSON API"))
         scriptOptions <- many (strOption (long "script-option" <> metavar "SCRIPT_OPTION" <> help "Pass option to Daml script interpreter"))
         shutdownStdinClose <- stdinCloseOpt
-        sandboxClassic <- SandboxClassic <$> switch (long "sandbox-classic" <> help "Deprecated. Run with Sandbox Classic.")
+        sandboxChoice <- sandboxChoiceOpt
+        cantonAdminApiPort <- sandboxPortOpt "canton-admin-api-port" "Port number for the canton admin API (--sandbox-canton only)"
```

Comment:
should we setup optparse-applicative to allow them only in combination? afaict in the CLI parser you do accept them even if you don’t use the canton sandbox.

Reply:
`mappend (Sum 1)`

Reply:
Changed this to only have the canton port options with --sandbox-canton. This is cleaner.

I also realize I had a bug in the option parser (I was using `switch` instead of `flag'`, which is what I need to the alternative work like intended). This was causing sandbox-classic to be the default.